ui/pages/settings/settings-tab/settings-tab.component.js

```diff
@@ -253,7 +256,8 @@ export default class SettingsTab extends PureComponent {
               <Typography
                 color={COLORS.TEXT_DEFAULT}
                 variant={TYPOGRAPHY.H7}
-                margin={[0, 0, 0, 3]}
+                margin={0}
+                marginBottom={3}
               >
                 {t('blockies')}
               </Typography>
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props